### ISSUE TITLE
Tutorial commands give error "cannot read key file at..."

### ISSUE THREAD

**Issue by okwme** [informalsystems/ibc-rs#1068]
## Crate
hermes 0.4.0

## Summary of Bug
Following instructions in the tutorial. We're on the [identifiers](https://hermes.informal.systems/tutorials/local-chains/identifiers.html) section. Running the following command:
```
hermes tx raw create-client ibc-1 ibc-0
```
gives the following error:
```
Jun 09 12:50:55.979  INFO ibc_relayer_cli::commands: using default configuration from '/Users/billy/.hermes/config.toml'
Error: tx error: error raised while creating client: failed while fetching the destination chain (ibc-1) signer: Keybase error: key store error: cannot read key file at '/Users/billy/.hermes/keys/ibc-1/keyring-test/testkey.json'
```
here's the tree of my `~/.hermes` directory:
```
> tree ~/.hermes
/Users/billy/.hermes
├── config.toml
└── keys
    ├── ibc-0
    │   └── keyring-test
    │       ├── testkey.json
    │       └── user2.json
    └── ibc-1
        └── keyring-test
            ├── testkey.json
            └── user2.json
```
And here's the tree of my `ibc-rs/data` directory:
```
> tree data -L 2
data
├── ibc-0
│   ├── config
│   ├── data
│   ├── keyring-test
│   ├── user2_seed.json
│   ├── user_seed.json
│   └── validator_seed.json
├── ibc-0.log
├── ibc-1
│   ├── config
│   ├── data
│   ├── keyring-test
│   ├── user2_seed.json
│   ├── user_seed.json
│   └── validator_seed.json
└── ibc-1.log
```
They look the same as the tutorial suggests, except the presence of the user-2 inside of the `~/.hermes` directory.

## Version
hermes 0.4.0

## Steps to Reproduce
Follow the tutorial. Both @jessicadosseh and I are having this error.

## Acceptance Criteria
Updates to the tutorial with instructions that succeed.

____

## For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate milestone (priority) applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by andynog** [comment]
Helping them troubleshoot the error

**Comment by adizere** [comment]
Confirm that @greg-szabo was encountering the same problem on a testnet with a Crypto.org Chain app.

The problem was manifesting as follows:

- use Hermes compiled from master to restore a key from a mnemonic to chain `ibc-6`:
```bash
> /opt/bin/hermes_0.4.0_post keys restore -m "....." --hd-path "m/44'/394'/0'/0/1" ibc-6
Success: Restored key 'wallet' (cro1x62wufc3w0pn7p37pgv9v2fyq6tnsw74mlsqnq) on chain ibc-6
```

- Use Hermes 0.4.0 (which does not include fix #1049) to load the new key and attempt to create a client

```bash
> "/opt/bin/hermes_0.4.0" create client ibc-6 ibc-5
Error: tx error: error raised while creating client: failed while fetching the destination chain (ibc-6) signer: Keybase error: key store error: cannot read key file at '/root/.hermes/keys/ibc-6/keyring-test/wallet.json'
```


- But using Hermes (post #1049) works

```bash
> "/opt/bin/hermes_0.4.0_post" create client ibc-6 ibc-5
Jun 10 09:17:30.651  INFO ibc_relayer_cli::commands: using default configuration from '/root/.hermes/config.toml'
Success: CreateClient(
    CreateClient(
        Attributes {
            height: Height {
                revision: 6,
                height: 51,
            },
            client_id: ClientId(
                "07-tendermint-0",
            ),
            client_type: Tendermint,
            consensus_height: Height {
                revision: 5,
                height: 12364,
            },
        },
    ),
)
```

**Comment by romac** [comment]
We should update the tutorial to mention that one needs to check out the tag corresponding to the current version before running the `dev-env` script.
